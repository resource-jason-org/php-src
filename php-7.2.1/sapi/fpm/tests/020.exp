[%s] NOTICE: fpm is running, pid %d
[%s] NOTICE: ready to handle connections
Ping ok
File php-fpm.acc.tmp exists
File php-fpm.slw.tmp exists
[%s] NOTICE: Terminating ...
[%s] NOTICE: exiting, bye-bye!
127.0.0.1 -  %s "GET /ping" 200