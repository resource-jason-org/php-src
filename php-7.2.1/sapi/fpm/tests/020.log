
---- EXPECTED OUTPUT
[%s] NOTICE: fpm is running, pid %d
[%s] NOTICE: ready to handle connections
Ping ok
File php-fpm.acc.tmp exists
File php-fpm.slw.tmp exists
[%s] NOTICE: Terminating ...
[%s] NOTICE: exiting, bye-bye!
127.0.0.1 -  %s "GET /ping" 200
---- ACTUAL OUTPUT
sh: 1: -F: not found

Warning: fsockopen(): unable to connect to 127.0.0.1:9008 (Connection refused) in /home/php-src/php-7.2.1/sapi/fpm/tests/fcgi.inc on line 255
Ping error
File php-fpm.acc.tmp missing
File php-fpm.slw.tmp missing

Warning: readfile(/home/php-src/php-7.2.1/sapi/fpm/tests/php-fpm.acc.tmp): failed to open stream: No such file or directory in /home/php-src/php-7.2.1/sapi/fpm/tests/020.php on line 47
---- FAILED
