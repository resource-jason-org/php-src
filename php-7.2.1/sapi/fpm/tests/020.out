sh: 1: -F: not found

Warning: fsockopen(): unable to connect to 127.0.0.1:9008 (Connection refused) in /home/php-src/php-7.2.1/sapi/fpm/tests/fcgi.inc on line 255
Ping error
File php-fpm.acc.tmp missing
File php-fpm.slw.tmp missing

Warning: readfile(/home/php-src/php-7.2.1/sapi/fpm/tests/php-fpm.acc.tmp): failed to open stream: No such file or directory in /home/php-src/php-7.2.1/sapi/fpm/tests/020.php on line 47